src/blockrelay/graphene.cpp

```diff
@@ -43,15 +44,18 @@ CGrapheneBlock::CGrapheneBlock(const CBlockRef pblock,
     uint64_t grapheneSetVersion = 0;
 
     version = _version;
+    computeOptimized = _computeOptimized;
```

Comment:
Its better to do these simple copies in the initializer list so there's no chance that the variables are used before they are set.

Reply:
Done.